README.md

```diff
@@ -59,6 +67,47 @@ EXAMPLES
 
 _See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v0.3.0/src/commands/autocomplete/index.ts)_
 
+## `tart delete DELETEFILE`
```

Comment:
Should probably order the commands in the order they'd be used in. So init first then save, load, delete, publish, fetch, unpublish or something to that effect.